event-stream-processing/src/__snapshots__/node-css-parser.test.ts.snap

```diff
@@ -0,0 +1,74 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic - GET request: 2f015294-45f8-4235-949b-667236bea641 1`] = `
+Object {
+  "@timestamp": "2021-07-14T23:17:05.036Z",
+  "agent": Object {
+    "type": "fluentbit",
+  },
+  "azp": "chefs-frontend",
```

Comment:
This is the username (service client id) that comes from Keycloak and is one of the only ways we can see which customer is using a common service. for chefs (because it have a browser-based frontend calling the api, the username is 'chefs-frontend' but in, for example, our CHES (email service) this azp could be 'PHSA' or any specific program area refernce. so it would be good to have that as an easy filter on our metrics. 
let me know if that makes sense.. but we i can keep in the original log object for now if you prefer.

Reply:
we settled on `client.user.id`